go.mod

```diff
@@ -7,4 +7,5 @@ require (
 	github.com/gofrs/uuid v4.2.0+incompatible
 	github.com/mitchellh/mapstructure v1.4.3
 	github.com/stretchr/testify v1.7.0
+	golang.org/x/sys v0.0.0-20220712014510-0a85c31ab51e
```

Comment:
is there something less specific to rely on?

Reply:
Not sure what you mean by less-specific. If you mean, could we target a tag instead, these system packages don't appear to have tagged versions. If you look on the right side of the page here, there isn't anything in the Tagged Versions list. https://pkg.go.dev/golang.org/x/sys